wordle-web/app.vue

```diff
@@ -1,46 +1,112 @@
 <template>
   <NuxtLayout>
     <v-app>
-      <v-app-bar color="primary" :elevation="2">
-        <template v-slot:prepend>
-          <v-icon color="secondary"> mdi-book </v-icon>
-        </template>
+      <v-app-bar app color="primary" :elevation="2">
+        <v-app-bar-nav-icon @click.stop="drawer = !drawer"></v-app-bar-nav-icon>
+        
+        <v-toolbar-title>
+          <v-avatar>
+            <v-icon color = "secondary">mdi-book</v-icon>
+          </v-avatar>
+          WordleApp
+        </v-toolbar-title>
 
-        <v-app-bar-title>WordleApp</v-app-bar-title>
+        <v-spacer></v-spacer>
+
+        <v-btn icon @click="router.push('/')">
+          <v-icon>mdi-home</v-icon>
+        </v-btn>
+
+        <v-btn icon @click="router.push('/about')">
+          <v-icon>mdi-information</v-icon>
+        </v-btn>
+
+        <v-btn icon @click="openSettingsDialog">
+          <v-icon>mdi-cog</v-icon>
+        </v-btn>
 
-        <v-btn @click="router.push('/')">Home</v-btn>
-        <v-btn @click="router.push('/test')">Test</v-btn>
-        <v-btn icon="mdi-theme-light-dark" @click="toggleTheme" />
         <v-btn icon="mdi-help-circle" @click="showHelpDialog = true" />
         <HelpDialog v-model="showHelpDialog" />
+
+        <v-btn icon>
+          <v-icon>mdi-dots-vertical</v-icon>
+        </v-btn>
       </v-app-bar>
+
+      <v-navigation-drawer v-model="drawer" app>
+        <v-list>
+          <v-list-item link @click="router.push('/')">
+            <v-list-item-icon>
+              <v-icon>mdi-home</v-icon>
+            </v-list-item-icon>
+            <v-list-item-title>Home</v-list-item-title>
+          </v-list-item>
+          
+          <v-list-item link @click="router.push('/about')">
+            <v-list-item-icon>
+              <v-icon>mdi-information</v-icon>
+            </v-list-item-icon>
+            <v-list-item-title>About</v-list-item-title>
+          </v-list-item>
+        </v-list>
+      </v-navigation-drawer>
+
       <v-main>
         <NuxtPage />
       </v-main>
+
+      <v-dialog v-model="settingsDialog" max-width="500">
+        <v-card>
+          <v-card-title>Settings</v-card-title>
+          <v-card-text>
+            Choose your theme:
+            <br>
+            <v-btn icon="mdi-weather-sunny" @click="toggleTheme('light')" />
+            Light
+            <br>
+            <v-btn icon="mdi-moon-waning-crescent" @click="toggleTheme('dark')" />
+            Dark
+            <br>
+            <v-btn icon="mdi-pine-tree" @click="toggleTheme('forest')"/>
```

Comment:
Good job on the 2 themes, I really love how you change them through clicking the icons. Looks very nice!